wagtail/admin/urls/scheduled_pages.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import path
+
+from wagtail.admin.views.scheduled_pages import (
+    publish,
+    publish_all_scheduled,
+    publish_all_scheduled_confirm,
+)
+
+app_name = "wagtailadmin_scheduled_pages"
+
+urlpatterns = [
+    path("<int:page_id>/publish/", publish, name="publish"),
+    path("publish_all/", publish_all_scheduled, name="publish_all"),
+    path(
+        "publish_all/confirm/",
+        publish_all_scheduled_confirm,
+        name="publish_all_confirm",
+    ),
+]
```

Comment:
how about something like

```
scheduled-pages/publish/<int:page_id>/
scheduled-pages/publish/all/
scheduled-pages/publish/confirm/
```

I am thinking the latter could be used for the single page or all selected pages confirmation